client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -123,3 +126,11 @@ export default connectOptions(
 		);
 	}
 );
+
+export default connect(
+	( state, { siteId, tier } ) => {
+		return {
+			wpcomTier: hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES ) ? tier : 'free'
```

Comment:
Would using the [`isPressable()`](https://github.com/Automattic/wp-calypso/blob/1cdd7380bb457dd4653fad1f27a6a53065d64c24/client/state/sites/selectors.js#L158) selector make our intentions clearer here?

Also, we perhaps we should hardode `'all'` instead of using `tier`, to be clearer about what is happening?